lib/purgatory/purgatory.rb

```diff
@@ -44,9 +44,14 @@ def soul_with_changes
   end
 
   def approve!(approver = nil)
-    return false if approved? 
-    if soul_with_changes.save
-      soul.send(performable_method[:method],*performable_method[:args]) if performable_method
+    return false if approved?
+    success = nil
```

Comment:
So @dobsonken, @thisduck, @MarissaMac, I'm starting to get cold feet about this change. What if other people are using Purgatory and their purgatized method returns `false` or `nil`. This would be a breaking change for them. Do you think it's worth it?